### DISCUSSION TITLE
Unexpected pytest-pylint error when using github action

### DISCUSSION THREAD

**Discussion by jiyoonlim123** [community/community#25264] [Actions]
<p>Hello.<br>
I want to ask how to fix unexpected github action error when using pytest-pylint.</p>
<p>When I pushed, pytest-pylint in GitHub action occurs ‘arguments-differ’ error.</p>
<p>But I cannot reproduce this error in my local folder. (There is no bug in our code)<br>
Even when I do git clone and lint, no error occurred.</p>
<p>I want ask you about below items.</p>
<ol>
<li>How to fix it</li>
<li>Why this error occurs</li>
</ol>
<p>(I used pytest==6.1.2)</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="jiyoonlim123" data-post="1" data-topic="234516">
<div class="title">
<div class="quote-controls"></div>
 jiyoonlim123:</div>
<blockquote>
<p>When I pushed, pytest-pylint in GitHub action occurs ‘arguments-differ’ error.</p>
</blockquote>
</aside>
<p>Why a test might fail is impossible to say without a look at the full log and code, I’m afraid. If the repository is public you can link it here, and either way I recommend looking at any differences between your local environment and that on the Actions runner (think Python version, dependencies, paths, …).</p>

**Comment by jiyoonlim123** [comment]
<p>Thanks for your help.<br>
I found that there is a python library version diffrerence.<br>
When I specify the version of that library, it solved.</p>
